scripts/fix_wheel.py

```diff
@@ -0,0 +1,47 @@
+import os
+import sys
+import zipfile
+from glob import glob
+
+if __name__ == "__main__":
+    # Due to https://github.com/python-poetry/poetry/issues/3509 and Poetry
+    # assuming the tags, we have to change the wheel ourselves after build
+
+    # Get the file from the dist dir
+    dist_files = glob("dist/*.whl")
+    assert len(dist_files) == 1, "Should have single wheel file in dist"
```

Comment:
I'd avoid using assert here, these statements will be ignored if a user has `PYTHONOPTIMIZE=true` exported.

Reply:
Will do (didn't figure that'd be a concern for a build script, but I understand it since it's an env var).

Reply:
Done